Update objects.md (#756)
Codecademy/docs: content/javascript/concepts/objects/objects.md

person.lastName;
```

> Or using square brackets. You must use bracket notation if the property name has numbers, spaces, or special characters:

Reviewer: ```suggestion
Bracket notation must be used if the property name has numbers, spaces, or special characters:
```

For Docs entries, let's avoid using "you", "your(s)", etc.
